.github/workflows/release.yml

```diff
@@ -35,31 +39,28 @@ jobs:
           zip oee-simulators.zip image.tar cumulocity.json
```

Comment:
- not sure if that will work; where is the image.tar coming from?
- I'd expect a call to build.yml to create the binaries

Reply:
This step in build.yml produce image.tar;
      - name: Build Docker image
        run: |
          cd repo/simulators
          docker build -t oee-simulators .
          docker save -o image.tar oee-simulators
          docker rmi oee-simulators